spec/integration/filtering_spec.rb

```diff
@@ -6,8 +6,9 @@
 
   it 'prints a rerun command for shared examples in external files that works to rerun' do
     write_file "spec/support/shared_examples.rb", """
-      RSpec.shared_examples 'a failing example' do
-        example { expect(1).to eq(2) }
+      RSpec.shared_examples 'with a failing example' do
+        example { expect(1).to eq(2) } # failing
+        example { expect(2).to eq(2) } # passing
       end
```

Comment:
I'm sure you probably know this, but I don't really understand why the triple quotes are here, so I'll just point out that they're just concatenating an empty string to a normal double quoted string

```sh
$ ruby-parse -e '"""abc"""'
(dstr
  (str "")
  (str "abc")
  (str ""))
```